website/docs/docs/build/incremental-microbatch.md

```diff
@@ -280,7 +284,129 @@ For now, dbt assumes that all values supplied are in UTC:
 
 While we may consider adding support for custom time zones in the future, we also believe that defining these values in UTC makes everyone's lives easier.
 
-## How `microbatch` compares to other incremental strategies?
+## Parallel batch execution
+
+The microbatch strategy offers the benefit of updating a model in smaller, more manageable batches. 
+
+Parallel batch execution means that multiple batches are processed at the same time, instead of one after the other (sequentially) for faster processing of your microbatch models.  
+
+dbt automatically detects whether a batch can be run in parallel in most cases, which means you don’t need to configure this setting. However, the `concurrent_batches` config is available as an override (not a gate), allowing you to specify whether batches should or shouldn’t be run in parallel in specific cases.
+
+For example, if you have a microbatch model with 12 batches, you can execute those batches to run in parallel. Specifically they'll run in parallel limited by the number of [available threads](/docs/running-a-dbt-project/using-threads).
+
+### Prerequisites
+
+To enable parallel execution, you must:
+
+- Use a supported adapter:
+  - Snowflake
+  - Databricks
+  - More adapters coming soon!
+    - We'll be continuing to test and add concurrency support for adapters. This means that some adapters might get concurrency support _after_ the 1.9 initial release.
+    
+- Meet [additional conditions](#how-parallel-batch-execution-works) described in the following section.
+
+### How parallel batch execution works
+
+A batch can only run in parallel if all of these conditions are met:
+
+| Condition     |  Parallel execution   | Sequential execution|
+| ---------------| :------------------: | :----------: |
+| **Not** the first batch |  ✅         | -            |
+| **Not** the last batch  |  ✅         | -            |
+| [Adapter supports](#prerequisites) parallel batches | ✅  | -         |
+
+
+After checking for the conditions in the previous table &mdash; and if `concurrent_batches` value isn't set, dbt will intelligently auto-detect if the model invokes the [`{{ this }}`](/reference/dbt-jinja-functions/this) Jinja function. If it references `{{ this }}`, the batches will run sequentially since  `{{ this }}` represents the database of the current model and referencing the same relation causes conflict. 
+
+Otherwise, if `{{ this }}` isn't detected (and other conditions are met), the batches will run in parallel, which can be overriden when you set a value for `concurrent_batches`.
```

Comment:
```suggestion
Otherwise, if `{{ this }}` isn't detected (and other conditions are met), the batches will run in parallel, which can be overriden when you set a value for `concurrent_batches`. For more information, refer to [`concurrent_batches`](/reference/resource-properties/concurrent_batches)
```